ci/gpu/build.sh

```diff
@@ -8,69 +12,118 @@ function logger() {
   echo -e "\n>>>> $@\n"
 }
 
+# apt-get install libnuma libnuma-dev
+
 # Arg parsing function
 function hasArg {
     (( ${NUMARGS} != 0 )) && (echo " ${ARGS} " | grep -q " $1 ")
 }
 
 # Set path and build parallel level
 export PATH=/conda/bin:/usr/local/cuda/bin:$PATH
-export CUDA_REL=${CUDA//./}
+export PARALLEL_LEVEL=4
+export CUDA_REL=${CUDA_VERSION%.*}
 
 # Set home to the job's workspace
 export HOME=$WORKSPACE
 
-# Switch to project root; also root of repo checkout
-cd $WORKSPACE
-
 # Parse git describe
+cd $WORKSPACE
 export GIT_DESCRIBE_TAG=`git describe --tags`
 export MINOR_VERSION=`echo $GIT_DESCRIBE_TAG | grep -o -E '([0-9]+\.[0-9]+)'`
-
-# Enable NumPy's __array_function__ protocol (needed for NumPy 1.16.x,
-# will possibly be enabled by default starting on 1.17)
-export NUMPY_EXPERIMENTAL_ARRAY_FUNCTION=1
+export UCX_PATH=$CONDA_PREFIX
 
 ################################################################################
 # SETUP - Check environment
 ################################################################################
 
-logger "Get env..."
+logger "Check environment..."
 env
 
+logger "Check GPU usage..."
+nvidia-smi
+
 logger "Activate conda env..."
 source activate gdf
+conda install "cudf>=0.10" "dask-cudf>=0.10" "cudatoolkit=$CUDA_REL" \
+              "dask>=2.3.0" "distributed>=2.3.2" "numpy>=1.16" -c rapidsai-nightly
```

Comment:
We shouldn't do this. These packages should all be installed by CI, but I think there's some issue currently with it, we should have ops to check.

Reply:
True, but until then I think we should do as in ucx-py and fix the versions?